urdf_root: data/urdfs
# gripper configuration, values should match the URDF
gripper:
  urdf_path: data/urdfs/panda/hand.urdf
  finger_depth: 0.05
  max_opening_width: 0.08
  T_tool0_tcp: {rotation: [0.0, 0.0, 0.0, 1.0], translation: [0.0, 0.0, 0.04]}
